tortoise.py

```diff
@@ -2,19 +2,36 @@
 
 
 def main():
-    
+    VALID_ARGS = ['tiny.tortoise', 'medium.tortoise', 'large.tortoise']
+
+    FILENAME = get_tortoise_file(sys.argv, VALID_ARGS)
+    if not '.tortoise' in FILENAME:
+        print(FILENAME)
```

Comment:
This isn't a feature we've discussed, but I think it's okay to leave it in. However, we would want a better error message to explain the problem to the user.